openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/CriticalCmdDialog.vue

```diff
@@ -0,0 +1,263 @@
+<!--
+# Copyright 2024 OpenC3, Inc.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# This file may also be used under the terms of a commercial license
+# if purchased from OpenC3, Inc.
+-->
+
+<template>
+  <v-dialog :persistent="persistent" v-model="show" width="600">
+    <v-card>
+      <v-system-bar>
+        <v-spacer />
+        <span> Waiting for Critical Command Approval </span>
+        <v-spacer />
+      </v-system-bar>
+
+      <v-card-title> {{ cmdString }} </v-card-title>
+      <v-card-subtitle>
+        User: {{ cmdUser }} - UUID: {{ uuid }}
+      </v-card-subtitle>
+      <v-card-text>
+        <v-container fluid>
+          <v-row v-if="!canApprove">
+            <v-text-field
+              v-model="username"
+              label="Username"
+              data-test="username"
+            />
+          </v-row>
+          <v-row v-if="!canApprove">
+            <v-text-field
+              v-model="password"
+              type="password"
+              label="Password"
+              data-test="password"
+            />
+          </v-row>
+          <v-row>
+            <v-spacer />
+            <v-btn
+              type="submit"
+              @click.prevent="reject"
+              class="mx-2"
```

Comment:
Add `outlined` to match existing dialogs

Reply:
I had this originally, but it didn't respect disabled (looked exactly the same), so I dropped it.